webservices/schemas.py

```diff
@@ -461,14 +461,7 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
             'contributor_employer_text',
             'contributor_occupation_text',
         ),
-        'relationships': [
-            Relationship(
-                models.ScheduleA.committee,
-                models.CommitteeHistory.name,
-                'committee_name',
-                1
-            ),
-        ],
+
```

Comment:
The committee relation is there, so I can simply add that back in.  When I was coding this I forgot what functionality this provided.  It should work just as the processed resources do.